### PR TITLE
Add system.make file for cosma

### PR DESCRIPTION
**Pull request by tkoskela**
Cosma is a HPC system in the University of Durham https://www.durham.ac.uk/departments/academic/physics/cosma/ It has a wide range of profiling tools available due to the performance analysis course ran by Tobias Weinzierl

### CODE REVIEW

**Comment by tkoskela** [comment]
This is ready for review, despite me making a mess of it on GitHub

File: src/system/system.cosma.make [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# system.make for cosma8 (apr 2023)
+# For user docs see https://cosma.readthedocs.io/en/latest/
+# load these modules
+# module load intel_comp/2022.3.0 compiler mpi mkl
```
Comment (ilectra) [inline src/system/system.cosma.make:4]: We should probably remove comments about modules to load, unless those comments are kept up-to-date...

File: src/system/system.cosma.make [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# module load fftw/3.3.10cosma8
+
+
+# Set compilers
+#FC=scorep --user mpif90
```
Comment (ilectra) [inline src/system/system.cosma.make:9]: Should remove `scorep` stuff as well.

**Review by ilectra — COMMENTED** [review]
Let's make sure this makefile works first, I'm not clear on how far you got today...

**Comment by tkoskela** [comment]
> Let's make sure this makefile works first, I'm not clear on how far you got today...

It worked for me, let me know when you've tested it

File: src/system/system.cosma.make (hunk 2 above)
Comment (tkoskela) [reply src/system/system.cosma.make]: done!

**Review by davidbowler — APPROVED** [review]

**Pull request merged**